packages/cyberstorm/src/components/FilterItemList/FilterItemList.tsx

```diff
@@ -62,10 +62,10 @@ export function FilterItemList(props: FilterItemListProps) {
   });
 
   return (
-    <>
+    <div className={styles.root}>
       <div className={styles.filterListHeader}>Categories</div>
-      {filters}
-    </>
+      <div>{filters}</div>
```

Comment:
I'm generally not a fan of empty divs. Is there a function for having this?

Reply:
Structure for the gap between "Categories" word and the list of filters.